### PR TITLE
copy node modules dir path into env

### PR DESCRIPTION
**Pull request by lillianzhang331**
In order to make node modules available downstream, copy the path into the build ENV for subsequent buildpacks.

### CODE REVIEW

File: buildpacks/nodejs/lib/build.sh [hunk 1]
```diff
@@ -162,6 +162,15 @@ set_node_env() {
 	fi
 }
 
+set_node_modules_env() {
+	local layer_dir=$1
+
+	mkdir -p "${layer_dir}/env"
+	if [[ ! -s "${layer_dir}/env/NODE_MODULES_ENV" ]]; then
+		echo "$(cd "$(dirname "${layer_dir}/node_modules")"; pwd -P)/$(basename "${layer_dir}/node_modules")" >>"${layer_dir}/env/NODE_MODULES_ENV"
```
Comment (lillianzhang331) [inline buildpacks/nodejs/lib/build.sh:170]: @danielleadams is this the right way to find path of node modules and copy it into NODE_MODULES_ENV?

File: buildpacks/nodejs/lib/build.sh [hunk 2]
```diff
@@ -162,6 +162,18 @@ set_node_env() {
...
 
+set_node_modules_env() {
+	local layer_dir=$1
+
+	mkdir -p "${layer_dir}/env"
+	if [[ ! -s "${layer_dir}/env/NODE_MODULES_ENV" ]]; then
+		echo "$(
+			cd "$(dirname "${layer_dir}/node_modules")"
+			pwd -P
+		)/$(basename "${layer_dir}/node_modules")" >>"${layer_dir}/env/NODE_MODULES_ENV"
```
Comment (danielleadams) [inline buildpacks/nodejs/lib/build.sh:173]: Can we rename the env var to `NODE_MODULES_PATH`? Otherwise it looks good!

**Review by danielleadams — APPROVED** [review]
🚀

File: buildpacks/nodejs/lib/build.sh (hunk 1 above)
Comment (danielleadams) [reply buildpacks/nodejs/lib/build.sh]: I think you may be able to do `echo "${layer_dir}/node_modules"`

**Pull request merged**